covid_graphs/covid_graphs/scripts/country_dashboard.py

```diff
@@ -77,53 +82,64 @@ def update_graph(graph_type_str, country_short_name):
+            decay given by:
+
+            * <em>N</em>(<em>t</em>) = (<em>A</em>/<em>T</em><sub><em>G</em></sub>) ⋅
+              (<em>t</em>/<em>T</em><sub><em>G</em></sub>)<sup>α</sup> /
+              e<sup><em>t</em>/<em>T</em><sub><em>G</em></sub></sup>
+
+            Where:
+
+            * <em>t</em> is time in days counted from a country-specific "day one"
+            * <em>N(t)</em> is the number of active cases (cumulative positively tested minus recovered and deceased)
+            * <em>A</em>, <em>T<sub>G</sub></em> and <em>α</em> are country-specific parameters
+
+            They made two predictions, on [March 30](/covid19/predictions/mar29) (for 7 countries)
+            and on [April 12](/covid19/predictions/apr11) (for 23 countries), each based on data available until
+            the day before. The first prediction assumed a common growth parameter <em>α</em> = 6.23.
+
             ### References
             * [Polynomial growth in age-dependent branching processes with diverging
               reproductive number](https://arxiv.org/abs/cond-mat/0505116) by Alexei Vazquez
             * [Fractal kinetics of COVID-19 pandemic]
               (https://www.medrxiv.org/content/10.1101/2020.02.16.20023820v2.full.pdf)
               by Robert Ziff and Anna Ziff
             * Unpublished manuscript by Katarína Boďová and Richard Kollár
-            """),
-        dcc.Markdown(f"""
-            ### Notes about the graphs
-
-            * Dashed lines are the predictions, solid red lines are the real active
-              cases. Black dotted lines mark the predicted maximums.
-            * We've now added new prediction made on 2020-04-13 by the same authors. There's a
-              second dashed line for Italy, Spain, USA and Germany.
-            * France has been excluded, since they [screwed up daily data reporting]({france_link}).
+            * March 30 predictions: [Facebook post]({mar30_prediction_link})
+            * April 13 predictions: Personal communication
+
+            ### Legend
             """,
-                     dangerously_allow_html=True)
+            dangerously_allow_html=True,
+        ),
+
+        # TODO: Include these?
+        # ### Notes about the graphs
+        # * France has been excluded, since they [screwed up daily data reporting]({france_link}).
+
+        html.Ul(
+            children=[
+                html.Li("Solid line is prediction"),
+                html.Li("Dashed line marks the culmination of the prediction"),
+                html.Li("Red line is observed number of active cases"),
+                html.Li(
+                    children=[
+                        "Data available until the date of prediction is in ",
+                        html.Span('light green zone', style={'background-color': 'lightgreen'}),
+                    ]
+                ),
+            ]
+        )
```

Comment:
I think this would read better as Markdown (you can include HTML for the span using `dangerously_allow_html=True`.

Reply:
I tried. It did not work for me, no idea why.